tests/importer/model_flattening.cpp

```diff
@@ -20,6 +20,16 @@ limitations under the License.
 
 #include "test_utils.h"
 
+TEST(ModelFlattening, invalidInput)
+{
+    auto importer = libcellml::Importer::create();
+
```

Comment:
Remove empty line.

Reply:
I like my blank line here it separates the declaration from the implementation.

Reply:
Isn't `auto flatModel = ...` a declaration?

Reply:
No it is the result of a call to the importer.